public/src/pages/main/Header/Header.tsx

```diff
@@ -14,7 +18,10 @@ export const Header: FC = () => {
   };
 
   return (
-    <header id="header" className="fixed-top ">
+    <header
+      id="header"
+      className={`fixed-top ${props.onInnerPage ? 'header-inner-pages' : null}`}
```

Comment:
`null` will be converted `null` string, e.g. `fixed-top null`

```suggestion
      className={`fixed-top ${props.onInnerPage ? 'header-inner-pages' : ''}`}
```